### DISCUSSION TITLE
service token permissions greyed out

### DISCUSSION THREAD

**Discussion by tuannguyen-qz** [planetscale/discussion#622] [Bug Reports]
I have admin role on the database that I'm trying to provision permissions for a service token. The `connect_production_branch` permission is greyed out, so I can't use the service token for connecting to branch `main`.

![Screenshot 2024-03-15 at 11 18 08 AM](https://github.com/planetscale/discussion/assets/141171454/2bf50d50-3b77-4eca-b31b-1c01f63ae50a)

**Comment by orware** [comment]
Hello @tuannguyen-qz,

These can be a little difficult to troubleshoot without further specifics.

If you don't mind, please write in over at https://support.planetscale.com/ with your relevant organization and database name and I can go ahead and help take a closer look!
